m4opt/missions/_lsst/__init__.py

```diff
@@ -51,13 +51,16 @@
 )
 lsst.__doc__ = r"""LSST, the Legacy Survey of Space and Time.
 
-`LSST <https://rubinobservatory.org/>`_ is a ground-based optical survey telescope 
-located in Chile as part of the Vera C. Rubin Observatory. It is designed 
-to conduct a 10-year survey of the southern sky with a large field of view 
-to detect transient events, including potential gravitational wave counterparts 
-(:footcite `2019ApJ...873..111I`).
+`LSST <https://rubinobservatory.org/>`_ is a ground-based optical survey
```

Comment:
Simonyi Survey Telescope is the telescope name, so good to include.

Reply:
Done.